src/pretix/control/forms/orders.py

```diff
@@ -167,6 +167,12 @@ class CancelForm(ForceQuotaConfirmationForm):
         initial=True,
         required=False
     )
+    comment = forms.CharField(
+        label=_('Comment (will be sent to the user)'),
+        help_text=_('Will be included in the notification email when the respective placeholder is present in the '
+                    'configured email text.'),
+        required=False,
+    )
```

Comment:
Should we make this a TextArea perhaps? I do see the advantage of a CharField (Keep you message simple and on point), but I could also see the advantage of a bigger text input area.

Second: Perhaps we should save the last message for easier recall if you are doing mass-cancellations?

Reply:
> Should we make this a TextArea perhaps? I do see the advantage of a CharField (Keep you message simple and on point), but I could also see the advantage of a bigger text input area.

First I wanted to agree and changed the widget, then I remembered that linebreaks and markdown don't always behave like users expect and unless we want to go through all that trouble of implementing a preview mode etc. it will be less error-prone with we stick to a single-line text (it's still not limited in length). I think we can leave it like this for now and always change it later if it turns out to be a problem in the real world.

> Second: Perhaps we should save the last message for easier recall if you are doing mass-cancellations?

Hmm, could be useful but is something we don't do anywhere else and if users do not expect it, they might not notice it and accidentally send the wrong comment to a user, so I think better not?